alis.sh

```diff
@@ -1552,6 +1558,19 @@ function desktop_environment_openbox() {
     arch-chroot /mnt systemctl enable lightdm.service
 }
 
+function desktop_environment_dusk() {
+    #dusk-git in AUR - set in alis-packages.conf
+    pacman_install "dmenu xterm lightdm lightdm-gtk-greeter xorg-server"
+    arch-chroot /mnt systemctl enable lightdm.service
+}
+
+function desktop_environment_leftwm() {
+    #leftwm-git in AUR - set in alis-packages.conf
+    #leftwm-theme-git in AUR - set in alis-packages.conf
+    pacman_install "dmenu xterm lightdm lightdm-gtk-greeter xorg-server"
```

Comment:
This is the same that for dusk. Should include leftwm, at least reading the wiki.